Apsitvarkom.UnitTests/DataAccess/PollutedLocationDTODatabaseRepositoryTests.cs

```diff
@@ -0,0 +1,194 @@
+﻿using Apsitvarkom.DataAccess;
+using Apsitvarkom.Models;
+using AutoMapper;
+using Apsitvarkom.Models.Mapping;
+using Moq;
+using MockQueryable.Moq;
+using System.Globalization;
+using static Apsitvarkom.Models.Enumerations;
+
+namespace Apsitvarkom.UnitTests.DataAccess;
+
+public class PollutedLocationDTODatabaseRepositoryTests
+{
+    private IMapper m_mapper = null!;
+    private Mock<IPollutedLocationContext> m_mockContext = null!;
+
+    [SetUp]
+    public void SetUp()
+    {
+        var config = new MapperConfiguration(cfg =>
+        {
+            cfg.AddProfile<PollutedLocationProfile>();
+        });
+        config.AssertConfigurationIsValid();
+        m_mapper = config.CreateMapper(); 
+        m_mockContext = new Mock<IPollutedLocationContext>();
+    }
+
+    #region Constructor tests
+    [Test]
+    public void PollutedLocationDTODatabaseRepositoryConstructor_HappyPath() =>
+        Assert.DoesNotThrow(() => new PollutedLocationDTODatabaseRepository(m_mockContext.Object, m_mapper));
+    #endregion
+
+    #region GetAllAsync tests
+    [Test]
+    public async Task GetAllAsync_DbContextHasNoInstances_EmptyListReturned()
+    {
+        var dbInstances = new List<PollutedLocation>();
```

Comment:
Nit: I'd prefer a more descriptive name such as `dbRows` since I am not sure what `dbInstances` refer to here without seeing how it's used